src/translations/nbTranslations.json

```diff
@@ -1403,7 +1405,19 @@
         "you_can_publish_metadata": "Beskrivelsen (metadata) av forskningsresultatet blir offentlig synlig når du velger \"$t(registration.public_page.tasks_panel.publish_registration)\".",
         "you_can_publish_metadata_files_info": "Når fil(er) og valgt lisens er godkjent av kurator blir de offentlig tilgjengelig."
       },
-      "validation_errors": "Valideringsfeil"
+      "duplicate_warning_modal": {
+        "headline": "Resultatet kan være en duplikat",
+        "publication_already_exists": "Det finnes allerede en publikasjon med samme tittel, år og kategori som din registrering.",
+        "dont_create_duplicates": "Du skal ikke opprette duplikater i NVA.",
+        "check_if": "Sjekk om",
+        "this_publication": " denne publikasjonen ",
+        "is_the_same_that_you_have_registered": "er det samme resultatet som du har registrert.",
```

Comment:
Mulig å løse dette med en nøkkel og bruke `<0>`-interpolering etc i stedet? Om andre språk skal ha annen setningsoppbygning vil det ellers bli trøbbel her. Ser ut som minst 3 av nøklene her er direkte avhengige av hverandre, så syns det gir mening om det slås sammen til en

Reply:
Bra innspill, men det blir ikke mulig på grunn av at deler av teksten skal være en <Link> med et ikon. Jeg tror det skal funke med kun engelsk og nynorsk, og spørsmålet er om vi skal fikse dette nå for potensielle problemer i fremtiden (husk YAGNI). Det jeg eventuelt kan gjøre er å bruke mindre deskriptive navn på variablene, altså navn som "text_before_link", "link_text" og "text_after_link". Alle språk vil jo ha denne inndelingen. Jeg brukte dette først, men endret det siden deres navn var veldig deskriptive for hva som var teksten.

Reply:
Husk at oversetter bare oversetter hver enkelt nøkkel hver for seg, så om jeg som oversetter hadde sett `"this_publication": " denne publikasjonen "` i lokalise hadde jeg nok tenkt at det var feil og bare skrevet "This publication" på engelsk, siden jeg ikke kan vite at denne avhenger av andre oversettelser før eller etter, og hvilke keys det eventuelt er. Ikon kan sendes inn som parameter på samme måt som Link ()med litt småskummel `<0>`-syntaks 😅), så om dette hadde vært en key kunne oversetter bestemt teksten på avsnittet via en key i stedet for å finne alle keys som hører sammen. Noe ala dette:

```
<Trans
  t={t}
  i18nKey="registration.public_page.duplicate_warning_modal.check_duplicate"
  components={[
    <MuiLink
      component={Link}
      target="_blank"
      data-testid={dataTestId.registrationLandingPage.duplicateRegistrationModal.duplicateRegistrationLink}
      to={getRegistrationLandingPagePath(duplicateId)}
      sx={{
        display: 'inline-flex',
        alignItems: 'center',
        gap: '0.25rem',
        mr: '0.25rem',
        color: 'primary.light',
      }}>
      <OpenInNewOutlinedIcon fontSize="small" />
    </MuiLink>,
  ]}
/>

// nbTranslations.json
"check_duplicate": "Du skal ikke opprette duplikater i NVA. Sjekk om <0>denne publikasjonen <0></0></0> er det samme resultatet som du har registrert."
```

Reply:
beste forklaringen jeg har på `<0>` er å se på det som indekser man sender inn som prop i `components`. Ytterste `<0>...</0>` er første indeks, altså `<MuiLink>`, mens `<0></0>` inni der er første child av `MuiLink` igjen, altså ikonet. På den måten kan oversetter endre tekster selv uten å måtte finne andre tekster som brekker som følge av det

Reply:
Prøvde det nå med koden din og det funka bra! Godt jobba, jeg er enig i at dette ble mye bedre og har mye mindre sjanse for oversettingsfeil.